src/main/java/com/dataguise/saas/dto/SourcesDetails.java

```diff
@@ -0,0 +1,97 @@
+package com.dataguise.saas.dto;
+
+import java.sql.Timestamp;
+import java.util.List;
+
+public class SourcesDetails {
+    private int sourceId;
+    private String sourceName;
+    private String sourceType;
+    private String sourceModule;
+    private String sourceAgent;
+    private String sourceHostName;
+    private Integer sourcePortNumber;
+    private List<String> sourceDatabases;
+    private List<String> sourceSchemas;
+
+    public Timestamp getCreatedAt() {
+        return createdAt;
+    }
+
+    public void setCreatedAt(Timestamp createdAt) {
```

Comment:
**Reporter**: Checkstyle    
    **Rule**: com.puppycrawl.tools.checkstyle.checks.coding.HiddenFieldCheck
    **Severity**: ERROR
    **File**: /work/src/main/java/com/dataguise/saas/dto/SourcesDetails.java L21
    
    &#39;createdAt&#39; hides a field.
    
*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1691487478>*